CMakeLists.txt

```diff
@@ -544,6 +546,15 @@ IF (PLUS_USE_PHILIPS_3D_ULTRASOUND)
   ENDIF()
 ENDIF()
 
+IF(PLUS_USE_OPTICAL_MARKER_TRACKING)

```

Comment:
The optical marker tracking shouldn't be dependent on the real sense SDK. In theory you can use it with any imaging device.